### ISSUE TITLE
  [ErrorException]   Declaration of BackupManager\Laravel\Laravel55Compatibility::table($headers   , $rows, $style = 'default') should be compatible with Illuminate\Console\C   ommand::table($headers, $rows, $tableStyle = 'default', array $columnStyles    = Array)

### ISSUE THREAD

**Issue by 4sh3** [backup-manager/laravel-4#93]
Error on Laravel55Compatibility, please update the solution
```

  [ErrorException]
  Declaration of BackupManager\Laravel\Laravel55Compatibility::table($headers
  , $rows, $style = 'default') should be compatible with Illuminate\Console\C
  ommand::table($headers, $rows, $tableStyle = 'default', array $columnStyles
   = Array)


```
Error on composer update:

```
Writing lock file
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postUpdate
> php artisan optimize


  [ErrorException]
  Declaration of BackupManager\Laravel\Laravel55Compatibility::table($headers
  , $rows, $style = 'default') should be compatible with Illuminate\Console\C
  ommand::table($headers, $rows, $tableStyle = 'default', array $columnStyles
   = Array)


Script php artisan optimize handling the post-update-cmd event returned with error code 1

```

Solution: Change the Laravel55Compatibility.php line 17:

```
public function table($headers, $rows, $style = 'default', array $columnStyles = []) {

```

**Comment by HiradRoshandel** [comment]
I'm having the same issue. @4sh3 's solution fixes it. It would be nice if it's pushed to the release

**Comment by ShawnMcCool** [comment]
Will merge a PR if one is submitted.

**Comment by asivaneswaran** [comment]
Running in the same issue, any ETA?

**Comment by nafiesl** [comment]
Use **dev-master** for temporary solution (before it tagged). The PR is already merged to master #91.

**Comment by ShawnMcCool** [comment]
New version has been tagged.
